src/recommendations/setting-cookie.md

```diff
@@ -0,0 +1,25 @@
+---
+group: product-recommendations
+title: Handle Cookie Restrictions
+ee_only: True
+---
+
+When you deploy the `magento/product-recommendations` module to production, the module tracks user events. These events enable behavioral data collection as long as the [cookie restriction mode](https://docs.magento.com/user-guide/stores/compliance-cookie-restriction-mode.html) is disabled. If you have custom code to handle the cookie restriction mode on your site, make sure you update your code to look for and handle the `mg_dnt` cookie.
+
+-  Name of the cookie:
+
+   ```bash
```

Comment:
I don't think bash is correct in these examples. Probably php or config, but these don't look like actual commands. bash will render the command prompt here.